### ISSUE TITLE
rules[i].match.exec is not a function

### ISSUE THREAD

**Issue by andreispatarelu** [ueberdosis/tiptap#2026]
### What’s the bug you are facing?

I get rules[i].match.exec is not a function when trying to use with react

Uncaught TypeError: rules[i].match.exec is not a function
    at run (index.es.js:76)
    at Plugin.handleTextInput (index.es.js:51)
    at index.es.js:2778
    at EditorView.someProp (index.es.js:5011)
    at readDOMChange (index.es.js:2778)
    at DOMObserver.handleDOMChange (index.es.js:3388)
    at DOMObserver.flush (index.es.js:3301)
    at MutationObserver.DOMObserver.observer (index.es.js:3168)

### How can we reproduce the bug on our side?

Add tiptap and add Image extension, try to write something in the editor

### What did you expect to happen?

TiptapE

### Did you update your dependencies?

- [X] Yes, I’ve updated my dependencies to use the latest version of all packages.

### Are you sponsoring us?

- [ ] Yes, I’m a sponsor. 💖

**Comment by caparker** [comment]
I am getting basically the same error I think

index.es.js:76 Uncaught TypeError: r[d].match.exec is not a function
    at c (index.es.js:76)
    at I.handleTextInput (index.es.js:51)
    at index.es.js:2710
    at Un.someProp (index.es.js:4874)
    at pe (index.es.js:2710)
    at Pe.<anonymous> (index.es.js:3297)
    at Pe.flush (index.es.js:3210)
    at MutationObserver.<anonymous> (index.es.js:3077)

**Comment by GatekoNeko141** [comment]
Help me, i soo have the same problem! I have to use version 2 of vue 

![image](https://user-images.githubusercontent.com/85969733/138029523-e5e67301-55be-4edc-ab0b-79957a4f1998.png)

**Comment by philippkuehn** [comment]
@GatekoNeko141 Yes, you have the same problem because you also did not update your dependencies.
